### PR TITLE
Menu map

### PR DESCRIPTION
**Pull request by erorcun**

### CODE REVIEW

File: src/core/Camera.h [hunk 1]
```diff
@@ -23,6 +23,16 @@ enum
 #define CAM_ZOOM_TOPDOWN (4.0f)
 #define CAM_ZOOM_CINEMATIC (5.0f)
 
+#ifdef FREE_CAM // LCS values
```
Comment (aap) [inline src/core/Camera.h:26]: This needs to check bFreeCam somewhere as well.

Comment (erorcun) [reply src/core/Camera.h:26]: ok

File: src/core/Camera.cpp [hunk 2]
```diff
@@ -2956,11 +2968,23 @@ CCamera::SetZoomValueFollowPedScript(int16 dist)
...
+		case 2: m_fCarZoomValueScript = ((CVehicle*)Cams[ActiveCam].CamTargetEntity)->IsBoat() ? FREE_CAM_BOAT_ZOOM_3 : FREE_CAM_CAR_ZOOM_3; break;
+		default: m_fCarZoomValueScript = m_fCarZoomValueScript; break;
+		}
+	} else
+#endif
+	{
+		switch (dist) {
+		case 0: m_fCarZoomValueScript = 0.05f; break;
+		case 1: m_fCarZoomValueScript = 1.9f; break;
+		case 2: m_fCarZoomValueScript = 3.9f; break;
```
Comment (ShFil119) [inline src/core/Camera.cpp:2985]:
```
#define CAR_CAM_ZOOM_1 0.05f
#define CAR_CAM_ZOOM_2 1.9f
#define CAR_CAM_ZOOM_3 3.9f
```
from header

Comment (ShFil119) [reply src/core/Camera.cpp:2985]: btw here and above could be used enum

Comment (erorcun) [reply src/core/Camera.cpp:2985]: enum with floats? :D
btw CAR_CAM_ZOOM_1 things were leftover, thanks for reminding

Comment (erorcun) [reply src/core/Camera.cpp]: ok, but if you mean changing CAM_ZOOM_x into enum you can ask that to aap, that's not my code

Comment (ShFil119) [reply src/core/Camera.cpp]: I thought about switch cases

**Pull request merged**